src/braket/experimental/autoqasm/program/program.py

```diff
@@ -43,6 +43,24 @@ def _get_local() -> threading.local:
     return _local
 
 
+@dataclass
+class SerializationConfig:
```

Comment:
I agree that `SerializationConfig` in `to_ir()` makes better sense. As you point out, the two serialization options does not happen in serialization time. 
For `return_capture_to_bitvar`, users may  want `capture_v0` without return to align better with [example in openqasm spec](https://openqasm.com/language/openpulse.html#capture-instruction). But so far, I ony find Braket uses `capture_v0`. So it should not be an issue default to  return to bitvar.
For `simplfy_constants`, like one of your comment below suggests, it looks prettier when using `pi` instead of numbers. It also avoids rounding error.

Overall, I think we can hardcode the default options, and expose them later upon request.

Reply:
In terms of moving it from `aq.main` to `to_ir()`, the difficulty is `simplfy_constants` which needs to be triggered in program building time because of oqpy.

Reply:
Yes, I guess we can just hardcode `simplify_constants` as `False` for now. It's not a functional thing (except for rounding error), it's just a prettiness of the OQ3.